submissions/soccer.md

```diff
@@ -0,0 +1,8 @@
+* submitter: @nirajanrk
+* statement: I hereby agree that the facts I have submitted are my own, and that I grant a license for Pryaxis Limited, or designated entities by Pryaxis Limited, the ability to use this data in a way to promote future Pryaxis services.
+----
+* Known as @Soccer in United States, the sport is known as @Football in the rest of the world.
```

Comment:
Ambiguous reference to football, which doesn't exist in your post. You need to only reference subjects with at least 5 facts.